src/test/java/apicurito/tests/steps/VerificationSteps/OperationVerifications.java

```diff
@@ -0,0 +1,248 @@
+package apicurito.tests.steps.VerificationSteps;
+
+import static com.codeborne.selenide.Condition.attribute;
+import static com.codeborne.selenide.Condition.matchText;
+import static com.codeborne.selenide.Condition.text;
+
+import org.openqa.selenium.By;
+
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+
+import java.util.List;
+
+import apicurito.tests.steps.OperationSteps;
+import apicurito.tests.utils.slenide.CommonUtils;
+import apicurito.tests.utils.slenide.MainPageUtils;
+import apicurito.tests.utils.slenide.OperationUtils;
+import cucumber.api.java.en.Then;
+import io.cucumber.datatable.DataTable;
+
+public class OperationVerifications extends CollectorHelper {
```

Comment:
You are using the same selectors in multiple methods - why don't you create an inner method with all selectors and use them as we discussed previously?

Reply:
Oh yeah, I forgot about that. Maybe in the next PR? This is already too big.